balboa-common/src/main/java/com/socrata/balboa/metrics/data/impl/BufferedDataStore.java

```diff
@@ -49,13 +49,8 @@ public BufferedDataStore(DataStore underlying) {
     public BufferedDataStore(DataStore underlying, TimeService timeService) {
         this.underlying = underlying;
         this.timeService = timeService;
-        try {
-            Configuration config = Configuration.get();
-            AGGREGATE_GRANULARITY = Long.parseLong(config.getProperty("buffer.granularity"));
-        } catch (IOException e) {
```

Comment:
Typesafe Config doesn't throw IOExceptions. Additionally, a missing file or config parameter indicates a serious error in the application, so we should let that error propagate to the top level.